src/parachain/nomination.ts

```diff
@@ -136,14 +136,16 @@ export interface NominationAPI extends TransactionAPI {
     /**
      * @param vaultAccountId Id of nominated vault
      * @param collateralCurrencyId The currency towards whose issuance the nomination was made
+     * @param rewardCurrencyId The reward currency, e.g. kBTC, KINT, interBTC, INTR
      * @param nominatorId Id of user who nominated to one or more vaults
      * @returns The rewards a (possibly inactive) nominator has accumulated
      */
     getNominatorReward(
-        vaultAccountId: AccountId,
+        vaultId: AccountId,
```

Comment:
Minor note but since `vaultId` is a specific type now, I wouldn't give an `accountId`-type variable this name